template.yaml

```diff
@@ -60,11 +74,26 @@ objects:
                 value: "7077"
               - name: SPARK_MASTER_WEBUI_PORT
                 value: "8080"
+              - name: NSS_WRAPPER_PASSWD
```

Comment:
this makes sense to me, i'd like to test it and understand the usage a little more